citi-onboarding/iJunior-onepage
# Feature/frontend-aboutUs

## What I did

- Created the frontend of the "aboutUs" section
- Developed the card component
   - Created the component and its API connection
   - Inserted the component card in the AboutUs views
   - Used the "slick" library to build the carousel

| Mockup | Real |
|--------|------|
| <img width="632" alt="Screen Shot 2021-03-16 at 14 03 32" src="https://user-images.githubusercontent.com/52111341/122833919-199bdc80-d2c4-11eb-94f0-fcfca68be5ac.png"> | <img width="632" alt="Screen Shot 2021-03-16 at 14 03 32" src="https://user-images.githubusercontent.com/52111341/122834267-ac3c7b80-d2c4-11eb-8cde-cbc75e4a4a7e.png"> |


## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `git checkout develop`
   1. `git pull`
   1. `git checkout feature/frontend-aboutUs`
   1. `git pull origin feature/frontend-aboutUs`
   1. `yarn install`
   1. `code .`
   1. `yarn start`

1. In your browser go to http://localhost:3000

1. Check if the component looks just like the mockup in deferent screen sizes 

1. In VScode open `client/src/views/aboutUs and client/src/components/cardCarousel`

1. Read the changed files section on GitHub 

1. Test the things that you think are worth testing, even the ones that are not in this description 

&nbsp;
***
**Time spent:** 7h30min

**Resolves:** #<Frontend of the "About Us" section>
